roadmap.yaml

```diff
@@ -31,14 +31,23 @@ Data Engineering:
     - Caching
   Web Frameworks and API Development:
     Python:
-      - Flask
-      - Django
-      - FastAPI
+      Web Development and API:
```

Comment:
I separated Python frameworks and libraries.

Reply:
Now this section looks like:
```
Web Frameworks and API Development:
…
      Web Development and API:
```
Which caused redundancy. It’s data engineering roadmap and we can keep related topics with higher of granularity.

Do you agree with that Farid jan? In case of any disagreement feel free to share your opinion with us.

Reply:
OK I will remove them.